src/buildblock/ML_norm.cxx

```diff
@@ -1399,7 +1412,10 @@ void apply_geo_norm(FanProjData& fan_data, const GeoData3D& geo_data, const bool
                         
                     }
                 }
-    
+
+#ifdef STIR_OPENMP
+  #pragma omp parallel for schedule(dynamic)
```

Comment:
You can probably use collapse(2) in this case it should parallelise the inner loop as well

Reply:
cool, will give it a go!

Reply:
It will need to be guarded with OpenMP version as in other places, as Visual Studio doesn't support it.

Reply:
Done.

Reply:
Does it speed it up further?

Reply:
Not noticeably, to be honest. But I thought it can't hurt to use it.